app/controllers/change_requests_controller.rb

```diff
@@ -148,6 +223,7 @@ def reject
   private
 
   def persist_change(change_request)
+    # need to figure out what my object_id is
```

Comment:
For new objects (e.g. "insert" actions), you won't have an object ID yet. Creating the new object should automatically assign it an ID.

Reply:
If this comment isn't needed anymore, could you remove it?